MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MapboxApplication.java

```diff
@@ -40,10 +41,6 @@ private void setUpTileLoadingMeasurement() {
   }
 
   private void setUpCrashlytics() {
-    Fabric.with(this, new Crashlytics.Builder()
-        .core(new CrashlyticsCore.Builder()
-            .disabled(BuildConfig.DEBUG)
-            .build())
-        .build());
+    Fabric.with(this, new Crashlytics());
```

Comment:
Does this mean that we'll send reports from the debug versions as well? I think we should stick to release builds only.

Reply:
Unfortunately, setting this to debug starts a cascade of other build issues because of all of the various Firebase libraries that are now in the app.

The Crashlytics section of the Firebase console is set up to release vs. debug anyway https://console.firebase.google.com/project/mapbox-android-demo/crashlytics. I've set up CircleCI with the release version of the google-services.json file, so that release crashes are sent to the right dashboard.

Reply:
Looks like I found a workaround

https://github.com/mapbox/mapbox-android-demo/pull/1139#issuecomment-517469922